tests/functional/suite_test.go

```diff
@@ -148,8 +151,8 @@ var _ = BeforeSuite(func() {
 
 	watcherv1.SetupDefaults()
 
-	//err = (&watcherv1.Watcher{}).SetupWebhookWithManager(k8sManager)
-	//Expect(err).NotTo(HaveOccurred())
+	err = (&watcherv1.Watcher{}).SetupWebhookWithManager(k8sManager)
+	Expect(err).NotTo(HaveOccurred())
```

Comment:
you only registered one fo the webhooks but we need all of them to be here.

Reply:
Done